src/decanter/core/core_api/body_obj.py

```diff
@@ -144,29 +143,39 @@ def create(cls, **kwargs):
 class TSGroupBy(CoreBodyObj):
     """Time series group by."""
     @classmethod
-    @corex_obj(
-        required={'time_unit', 'regression_method', 'classification_method'})
+    @corex_obj(required=None)
     def create(cls, **kwargs):
         """Return TSGroupBy object with passed kwargs as attributes"""
+        at_least = {
```

Comment:
check requirements with decorator, corex_obj.

Reply:
Thanks, I'll fix that~